src/localisation/types.ts

```diff
@@ -37,6 +37,7 @@ enum Phrase {
   RecordBattlegroundsDescription,
   CaptureCursorDescription,
   MinKeystoneLevelDescription,
+  MinMopChallengeTierDescription,
```

Comment:
The naming of this key isn't quite right

Reply:
Yeah, i'll change this and remove MinMoP and name it only MoPChallengeMode